README.rst

```diff
@@ -657,14 +717,10 @@ port. Indeed, clients will use port 443 by default, whereas servers default to
 port 8448. Where these are different, we refer to the 'client port' and the
 'federation port'.
 
-The next most important thing to know is that using a reverse-proxy on the
-federation port has a number of pitfalls. It is possible, but be sure to read
-`Reverse-proxying the federation port`_.
-
-The recommended setup is therefore to configure your reverse-proxy on port 443
-to port 8008 of synapse for client connections, but to also directly expose port
-8448 for server-server connections. All the Matrix endpoints begin ``/_matrix``,
-so an example nginx configuration might look like::
+The recommended setup is therefore to configure your reverse-proxy on port
+443 to port 8008 of synapse for client connections, and port 8448 for
```

Comment:
this makes no sense.

I think the recommendation (if you're going to use a reverse-proxy) should be:

* set `no_tls: True` and remove the `tls` listeners from the synapse config
* have reverse proxy listen on both 443 and 8448, and forward both to port 8008.

However, this is starting to all get out of scope. For now I suggest: you remove this paragraph and just say "An example nginx configuration for forwarding client connections to synapse is::"